pkg/cmd/version/testdata/TestGetVersions-deployment_only_with_pipeline_installed.golden

```diff
@@ -0,0 +1,2 @@
+Client version: dev
+Pipeline version: unknown, pipeline controller may be installed in another namespace please use tkn version -n {namespace}
```

Comment:
I think we should tweak this error message, but I can put in a follow up pr. It shouldn't delay this pr.